### ISSUE TITLE
Chord with tied-into grace notes is screwy

### ISSUE THREAD

**Issue by worldwideweary** [Jojo-Schmitz/MuseScore#169]
### Issue type

Engraving bug

### Bug description

This thing has existed since somewhere in 3.x prior to PR#9000, if I remember correctly. It was corrected in transition though so it's not in 4.x, but is in 3.7:

### Steps to reproduce

1) Make a chord with more than one note on it
2) Make grace notes that tie into that chord

### Screenshots/Screen recordings

![MS3 7 - Tied Grace Notes screwed up](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/e73d76bb-2e6b-46cb-b611-afc3b26326c4)

### MuseScore Version

3.7

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

...

### Additional context

A naive way to fix this is to just pull out of all adjustments if the conditions are met. That's what I did in the past

For example, in `libmscore/tie.cpp`:`void TieSegment::adjustX()`

```cpp
       if (isNudged() || isEdited())
             return;
 
+      if (sn->noteType() != NoteType::NORMAL) {
+            if (en->chord() && en->chord()->notes().size() > 1) {
+                  return;
+                  }
+            }
+
```

Where not normal = practically grace notes and the end chord has more than one note on it, i just quit all further adjustments. Certainly there's another way to include some niceties of further adjustment, in the meantime that will work and give:

![MS3 7 - Tied Grace Notes better](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/a4e6ac66-e28b-4391-b71e-c55274676b90)

**Comment by Jojo-Schmitz** [comment]
Hmm, that's definitly not the way this got fixed for master

**Comment by Jojo-Schmitz** [comment]
Seems to happen also when the tie target is a single note

**Comment by worldwideweary** [comment]
Update:
Not really suggesting my workaround, but figured I'd mention it earlier. Actually now that you mention it, I've changed it a bit to reset the offsets to 0.0 and then give a little bit of a margin. I'll copy/paste some code image changes in case it's of any value to you, but again not necessarily suggesting it:

![1-code](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/7c425496-970a-4513-8dd9-595665ba41b2)

![2-code](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/3f276d26-8c75-41c7-baba-e0604c4b5577)

![3-code](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/149b30fa-13bd-427a-8fad-dce5e6e3955b)

Results will give something decent like
![4](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/2b26c67c-2535-42be-8888-daad75c565f3)
![5](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/d5ecb1a7-b498-4996-bccc-b2dc3baa9c0a)

**Comment by Jojo-Schmitz** [comment]
Grace notes after do have the same glitch, even after your non-suggestions ;-)

**Comment by Jojo-Schmitz** [comment]
This looks better though:
![image](https://github.com/Jojo-Schmitz/MuseScore/assets/1786669/7346360c-8d38-416e-a54c-8b393f793bf3)

**Comment by worldwideweary** [comment]
Resolved with #170

**Comment by Jojo-Schmitz** [comment]
Ah, yes, I forgot to close this, thanks
